zerver/lib/streams.py

```diff
@@ -227,7 +388,7 @@ def filter_stream_authorization(user_profile: UserProfile,
 
 def list_to_streams(streams_raw: Iterable[Mapping[str, Any]],
                     user_profile: UserProfile,
-                    autocreate: bool=False) -> Tuple[List[Stream], List[Stream]]:
+                    autocreate: bool = False) -> Tuple[List[Stream], List[Stream]]:
```

Comment:
Can you turn off your editor's auto-editing of code, and revert these changes?

Reply:
Okay